src/applications/appeals/995/validations/evidence.js

```diff
@@ -58,9 +58,18 @@ export const validateVaToDate = (errors, data) => {
   }
 };
 
+// Check if VA evidence object is empty
+export const isEmptyVaEntry = (checkData = {}) =>
+  [
+    checkData.locationAndName || '',
+    (checkData.issues || []).join(''),
+    Object.values(checkData.evidenceDates || {}).join(''),
```

Comment:
had an idea for this... instead of having to do `.join('')` multiple times, couldn't we spread these out?

```
[
    checkData.locationAndName || '',
    ...(checkData.issues || []),
    ...Object.values(checkData.evidenceDates || {}),
]
```

Just a thought, but probably just a nit. Not sure if there are any real benefits to either way

Reply:
True, that does look cleaner. Thanks!